tests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/connection/v3/OpenNewConnectionWizardTest.java

```diff
@@ -28,50 +27,44 @@
  * Test opening new connection dialog via tool item and context menu.
  * 
  * @author mlabuda@redhat.com
+ * @author adietish@redhat.com
  *
  */
-public class OpenNewConnectionShellTest {
+public class OpenNewConnectionWizardTest {
 	
 	OpenShiftExplorerView explorer = new OpenShiftExplorerView();
 	
 	@Test
-	public void testOpenConnectionShellViaLinkTest() {
+	public void shouldOpenConnectionWizardViaLinkTest() {
+		// given
+		ConnectionUtils.clearAll();
 		explorer.open();
-		
-		try {
-			new DefaultLink(OpenShiftLabel.TextLabels.NEW_CONNECTION).click();
-			// PASS
-		} catch (SWTLayerException ex) {
-			fail("New Connection shell cannot be opened via context menu.");
-		}
-		
-		verifyNewConnectionShellIsOpened();
+
+		// when
+		new DefaultLink(OpenShiftLabel.TextLabels.CREATE_CONNECTION).click();
+
+		// then
+		verifyNewConnectionWizardIsOpened();
 	}
 	
 	@Test
-	public void testOpenConnectionShellViaToolItemTest() {
+	public void shouldOpenConnectionWizardViaToolItemTest() {
+		// given
 		explorer.open();
 		
-		try {
-			new DefaultToolItem(OpenShiftLabel.Others.CONNECT_TOOL_ITEM).click();
-			// PASS
-		} catch (SWTLayerException ex) {
-			fail("New Connection shell cannot be opened via view tool item.");
-		}
-		
-		verifyNewConnectionShellIsOpened();
+		// when
+		new DefaultToolItem(OpenShiftLabel.Others.CONNECT_TOOL_ITEM).click();
+
+		// then
+		verifyNewConnectionWizardIsOpened();
 	}
 	
-	private void verifyNewConnectionShellIsOpened() {
-		Shell connectionShell = null;
-		try {
-			connectionShell = new DefaultShell(OpenShiftLabel.Shell.NEW_CONNECTION);
-		} catch (SWTLayerException ex) {
-			fail("New Connection shell was not opened.");
-		}
-		
+	private void verifyNewConnectionWizardIsOpened() {
```

Comment:
The problem with the previous approach is that they hide/swallow the exception and at best (in some cases) include the exception message in the fail message. IMHO if this was the better approach, the framework should do that, repeating this exception swallowing in every test seems like an indication to me that either we should drop this and unveil the root exception or have this done in red deer.
I dont get why dropping the swallowing wont result in a failed test? If an exception occurrs in a test, the test is failing, isnt it?